process-automation/latest/model/files.md

```diff
@@ -2,7 +2,7 @@
 title: Files
 ---
 
-Files can be uploaded and used within a project.
+Files can be uploaded and used within a project. The file can be an image or a script, and the script can contain a template for use within the modelling application.
```

Comment:
the file can be whatever the user wants, usually it is an image or a script

Reply:
adjusted, let me know what you think, thanks.